sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -1184,3 +1201,15 @@ func withUnmaskedPaths(unmaskPaths []string) oci.SpecOpts {
 		return nil
 	}
 }
+
+// Convert labels to map[string]string for containerd.WithPullLabels.
+// Labels are in the format of "key=value,key2=value2".
+func convertLabels(labels string) map[string]string {
+	labelsMap := make(map[string]string)
```

Comment:
This could use some error handling, it is possible a user will put malformed input here and this function shouldn't provide undefined values if that is the case. I think it should validate its input by checking that `labels` is not empty (and if so just return the empty map). It should also ensure it gets "good" splits on the `,` and `=`. I'm sort of focused on someone forgetting a value and doing `--labels="foo=bar,baz="` or `--labels="foo=bar,baz,fuzz"` and such.